nano/nano_node/daemon.cpp

```diff
@@ -49,6 +43,7 @@ void nano_daemon::daemon::run (boost::filesystem::path const & data_path, nano::
 	{
 		config.node.logging.init (data_path);
 		nano::logger_mt logger{ config.node.logging.min_time_between_log_output };
+		sigman.set_logger (&logger);
```

Comment:
This should be sigman.set_logger (logger) once the parameter is changed to a reference.